lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -475,6 +500,7 @@ def __init__(
         network_attachment_definitions_func: Callable[[], list[NetworkAttachmentDefinition]],
         network_annotations: list[NetworkAnnotation],
         container_name: str,
+        refresh_event: BoundEvent,
```

Comment:
Is there any reference explaining how to use `BoundEvent`? It's the first time I see it.
Why can't we just `nad_config_changed` event?

Reply:
BoundEvent is  an event bound to an object. https://ops.readthedocs.io/en/latest/_modules/ops/framework.html#BoundEvent.

Reply:
Yes, I found that too, but honestly, it means nothing to me :)
How is it mapped to an actual `NadConfigChangedEvent` event? And what is the benefit of using this over `on.nad_config_changed`?

Reply:
This is also first time that I am using BoundEvent and I do not see any benefit to define the type as BoundEvent. I am converting it to EventBase.